cmd/kubeadm/app/phases/certs/certs_test.go

```diff
@@ -472,11 +484,12 @@ func TestSharedCertificateExists(t *testing.T) {
 
 			// executes create func
 			ret, err := SharedCertificateExists(cfg)
-
 			switch {
-			case !test.expectedError && err != nil:
-				t.Errorf("error SharedCertificateExists failed when not expected to fail: %v", err)
-			case test.expectedError && err == nil:
+			case err != nil:
+				if agg, ok := err.(utilerrors.Aggregate); ok && len(agg.Errors()) != test.expectedErrors {
+					t.Errorf("SharedCertificateExists didn't failed with expected error numbers")
+				}
+			case err == nil && test.expectedErrors != 0:
 				t.Errorf("error SharedCertificateExists didn't failed when expected")
```

Comment:
```suggestion
				t.Errorf("error SharedCertificateExists didn't fail when expected")
```

^ there is also a typo in the old code

Reply:
:)